extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-postgresql/src/main/java/org/apache/guacamole/auth/postgresql/PostgreSQLEnvironment.java

```diff
@@ -47,6 +47,21 @@
      */
     private static final int DEFAULT_PORT = 5432;
 
+    /**
+     * The default defaultStatementTimeout (in seconds),
+     * if POSTGRESQL_DEFAULT_STATEMENT_TIMEOUT is not specified.
+     * Default to 0 (no timeout, property won't be set)
+     * https://mybatis.org/mybatis-3/configuration.html
```

Comment:
A couple things for this comment block:
* I'd avoid using the actual variable name (`defaultStatementTimeout`) and just make it read like "The default statement timeout..."
* We don't really link to outside documentation for any of the other properties, and I'm not sure it's worth starting for these two, so I'd remove the links to external sites.
* For the explanation for "Default to 0," something more verbose might be appropriate: "Setting this value to 0 (the default) will result in..."